### PR TITLE
Setup mongoDB and created storage folder

### PR DESCRIPTION
**Pull request by isha-gautam**

### CODE REVIEW

File: setupDB.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    {
+        user: "AnthroAdmin",
+        pwd: "password",
+
+        roles: [{ role: "userAdmin", db: "AnthroLinkDB" }]
+    }
+);
+
+db.createCollection("users");
+
```
Comment (rgpt) [inline setupDB.js:16]: Add tickets collection ?

File: .gitignore [hunk 2]
```diff
@@ -1,2 +1,4 @@
 node_modules/
+
```
Comment (rgpt) [inline .gitignore:2]: Remove Extra Line

File: setupDB.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+db.createUser(
+    {
+        user: "adminDB",
+        pwd: "password",
+
+        roles: [{ role: "userAdminAnyDatabase", db: "admin" }]
+    }
+);
+
+
```
Comment (rgpt) [inline setupDB.js]: Remove Extra Lines below.

**Review by rgpt — CHANGES_REQUESTED** [review]

File: setupDB.js (hunk 3 above)
Comment (isha-gautam) [reply setupDB.js]: I should leave a couple lines though, shouldn't I? The coding practices you've told me about before?

**Pull request merged**